packages/apps-config/src/api/spec/dotmog.ts

```diff
@@ -19,7 +19,8 @@ const definitions: OverrideBundleDefinition = {
           dna: 'Hash',
           genesis: 'BlockNumber',
           price: 'Balance',
-          gen: 'u64'
+          gen: 'u32',
+		  rarity: 'RarityType'
```

Comment:
```suggestion
          gen: 'u32',
		      rarity: 'RarityType'
```

Should fix linting, if not just run `yarn lint` locall.